astroquery/heasarc/core.py

```diff
@@ -250,11 +256,12 @@ def _parse_result(self, response, verbose=False):
 
         try:
             data = BytesIO(response.content)
-            return Table_read(data, hdu=1)
+            with u.add_enabled_aliases(UNIT_MAPPER):
+                return Table_read(data, hdu=1)
         except ValueError:
             try:
                 return self._fallback(response.text)
-            except Exception as e:
+            except Exception:
```

Comment:
this is a catch-all Exception.  I think we should keep this as
```suggestion
            except Exception as ex:
                print(f"Caught exception {ex}: using fallback")
```
because I don't like catching generic `Exceptions` (that can block even user-triggered control-c, for example).  It would be better to catch the _right_ exceptions, but without knowing what those are, I suggest we just be noisy instead.

Reply:
(this is also extremely minor and it's fine to ignore me)

Reply:
yeah, I agree, I have the same annoyance (on top of catching them but don't use the stored item), but didn't know what the expected exception would be from `_fallback()`. OTOH, here I clearly made this change quite late in the night without reading properly that `ex` is in fact used 🤦‍♀️

Reply:
> I don't like catching generic `Exceptions` (that can block even user-triggered control-c, for example)

`except:` does catch `KeyboardInterrupt`, but `except Exception:` does not.

Reply:
https://docs.python.org/3/library/exceptions.html#KeyboardInterrupt